### ISSUE TITLE
Compatibility Issue with ESM in SvelteKit

### ISSUE THREAD

**Issue by kyrregjerstad** [timomeh/tiptap-extension-code-block-shiki#2]
Hi, 
when importing `tiptap-extension-code-block-shiki` in a SvelteKit project, I encounter an error related to ESM module compatibility during SSR. The error message is:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module ./node_modules/.pnpm/shiki@1.10.0/node_modules/shiki/dist/index.mjs not supported.
Instead change the require of ./node_modules/.pnpm/shiki@1.10.0/node_modules/shiki/dist/index.mjs to a dynamic import() which is available in all CommonJS modules.
    at Module.<anonymous> (./node_modules/.pnpm/tiptap-extension-code-block-shiki@0.2.1_@tiptap+core@2.4.0_@tiptap+pm@2.4.0__@tiptap+extensio_3xnresjn5zzacdejhmqaqjnxlq/node_modules/tiptap-extension-code-block-shiki/dist/tiptap-extension-code-block-shiki.cjs:1:345) {
  code: 'ERR_REQUIRE_ESM'
}
Error [ERR_REQUIRE_ESM]: require() of ES Module ./node_modules/.pnpm/shiki@1.10.0/node_modules/shiki/dist/index.mjs not supported.
Instead change the require of ./node_modules/.pnpm/shiki@1.10.0/node_modules/shiki/dist/index.mjs to a dynamic import() which is available in all CommonJS modules.
    at Module.<anonymous> (./node_modules/.pnpm/tiptap-extension-code-block-shiki@0.2.1_@tiptap+core@2.4.0_@tiptap+pm@2.4.0__@tiptap+extensio_3xnresjn5zzacdejhmqaqjnxlq/node_modules/tiptap-extension-code-block-shiki/dist/tiptap-extension-code-block-shiki.cjs:1:345) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Steps to Reproduce:**

- Create a SvelteKit project.
- Install tiptap-extension-code-block-shiki and shiki.
- Import tiptap-extension-code-block-shiki in a Svelte component.
- Run the project with SSR enabled (the default option).

**Expected Behavior:**
The package should work seamlessly with SvelteKit's SSR without requiring dynamic imports or disabling SSR.

**Actual Behavior:**
The project fails to render due to the ESM module being required incorrectly.

**Temporary Workaround:**
Disabling SSR resolves the issue, but it's not an ideal solution.

Directly copying and running the code from the repository works fine, indicating a bundling issue.

**Comment by timomeh** [comment]
Ah, the most beloved error in node's history, `ERR_REQUIRE_ESM`.

Thanks for the great issue description, I really appreciate it. If you can open a PR, I'm happy to review and accept a fix.

**Comment by timomeh** [comment]
Thanks for the PR ❤️ released in v0.3.0
